esp8266-fastled-webserver/include/simplehacks/array_size2.h

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
SIMPLE FILE REVIEW

Enables safer `ARRAY_SIZE2` macro, which fails to compile if given a raw pointer.